src/pydata_sphinx_theme/__init__.py

```diff
@@ -501,6 +502,50 @@ def get_edit_url():
     context["theme_show_toc_level"] = int(context.get("theme_show_toc_level", 1))
 
 
+# -----------------------------------------------------------------------------
+# handle pygment css
+# -----------------------------------------------------------------------------
+def _get_styles(formatter, prefix):
+    """
+    Get styles out of a formatter, where everything has the correct prefix.
+    """
+
+    for line in formatter.get_linenos_style_defs():
+        yield f"{prefix} {line}"
+    yield from formatter.get_background_style_defs(prefix)
+    yield from formatter.get_token_style_defs(prefix)
+
+
+def get_pygments_stylesheet():
+    """
+    Generate the theme-specific pygments.css.
+    There is no way to tell Sphinx how the theme handles modes
+    """
+    light_formatter = HtmlFormatter(style="tango")
```

Comment:
Could this be configurable at the theme level? E.g. `html_theme_options.pygments_theme_light`, and `html_theme_options.pygments_theme_dark`.

Reply:
do I have access to theme-related variables in __init__.py ?